apps/api/src/schema/library-traffic.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from "@hono/zod-openapi";
+
+export const libraryTrafficQuerySchema = z.object({
+  locationName: z
+    .string()
+    .openapi({
+      param: { name: "locationName", in: "query" },
+    })
+    .optional(),
+});
+
+export const libraryTrafficSchema = z.object({
```

Comment:
we typically have descriptions on these fields; i argue the meaning here isn't 
perfectly obvious from just names and examples (even if i have a 75% idea) so you should add descriptions here